src/app/index.ts

```diff
@@ -13,18 +13,16 @@ import { promisify } from "util";
 import * as usageData from "office-addin-usage-data";
 import * as uuid from 'uuid/v4';
 import * as yosay from 'yosay';
-import * as yo from 'yeoman-generator';
+const yo = require("yeoman-generator");
```

Comment:
I tried to just do an import here but it caused all sorts of compile issues; hence I just did require.  Not happy with this but couldn't figure out how to make the import work